src/includes/getting-started-primer/java.jul.mdx

```diff
@@ -0,0 +1,7 @@
+<Note>
+
+_The `sentry-jul` library provides a [java.util.logging Handler](http://docs.oracle.com/javase/8/docs/api/java/util/logging/Handler.html) that sends logged exceptions to Sentry._
+
+</Note>
+
+Once this integration is configured you can _also_ use Sentry’s static API, [as shown on the usage page](/usage), in order to do things like record breadcrumbs, set the current user, or manually send events. The source can be found [on GitHub](https://github.com/getsentry/sentry-java/tree/master/sentry-jul).
```

Comment:
Seems like `/usage` won't work here since this is just an include

Reply:
probably it needs to be `/platforms/java/guides/{integration}/usage/`
or to find a way to direct to /usage/ but keeping the current platform, I guess @philipphofmann found out a workaround for this because of the Apple platform and its variations (iOS/watchOS, etc...)

Reply:
Yes, you can use a `<PlatformLink>`, here is one in action https://github.com/getsentry/sentry-docs/blame/dd814c38bd6b61af21ffd00364198d3d9f9cb6c0/src/platforms/common/enriching-events/attachments/index.mdx#L76

Reply:
Fixed, thanks!